optimize/backend/src/main/java/io/camunda/optimize/service/db/es/reader/DurationOutliersReaderES.java

```diff
@@ -720,7 +732,11 @@ private Map<String, Map<String, Long>> createVariableTermOccurrencesMap(
                                 AbstractMap.SimpleEntry::getKey,
                                 AbstractMap.SimpleEntry::getValue));
 
-                outlierVariableTermOccurrences.put(variableName, termOccurrences);
+                // We resolve this back to its original name in the case that it was sanitised
```

Comment:
```suggestion
                // We resolve this back to its original name in the case where it was sanitised
```

Reply:
not sure about this one since you're the native speaker, you can judge it best :-)